examples/speech-recognition/README.md

```diff
@@ -187,3 +203,176 @@ python run_speech_recognition_ctc.py \
+	--gradient_checkpointing \
+	--group_by_length \
+	--bf16 \
+	--overwrite_output_dir \
+	--do_train \
+	--do_eval \
+	--predict_with_generate \
+	--use_habana \
+	--use_hpu_graphs_for_inference
+```
+
+#### Single GPU Seq2Seq Inference
+
+The following example shows how to do inference with the [Whisper small](https://huggingface.co/openai/whisper-small) checkpoint on the Hindi subset of [librispeech_asr_demo](https://huggingface.co/datasets/librispeech_asr_demo) using 1 HPU devices in half-precision:
+
+```bash
+python run_speech_recognition_seq2seq.py \
+    --model_name_or_path="openai/whisper-small" \
+        --dataset_name="hf-internal-testing/librispeech_asr_demo" \
+        --gaudi_config_name="gaudi_config.json" \
+        --dataset_config_name="clean" \
+        --eval_split_name="validation" \
+        --max_steps="5000" \
+        --output_dir="./results/whisper-small-clean" \
+        --gradient_accumulation_steps="2" \
+        --per_device_eval_batch_size="16" \
+        --logging_steps="25" \
+        --learning_rate="1e-5" \
+        --warmup_steps="500" \
+        --evaluation_strategy="steps" \
+        --eval_steps="1000" \
+        --save_strategy="steps" \
+        --save_steps="1000" \
+        --generation_max_length="225" \
+        --preprocessing_num_workers="16" \
+        --length_column_name="input_length" \
+        --max_duration_in_seconds="30" \
+        --freeze_feature_encoder="False" \
+        --gradient_checkpointing \
+        --group_by_length \
+        --bf16 \
+        --overwrite_output_dir \
+        --do_eval \
+        --predict_with_generate \
+        --use_habana \
+		--use_hpu_graphs_for_inference
+```
+The following example shows how to do inference with the [Whisper small](https://huggingface.co/openai/whisper-small) checkpoint on the Hindi subset of [Common Voice 11](https://huggingface.co/datasets/mozilla-foundation/common_voice_11_0) using 1 HPU devices in half-precision:
+
+```bash
+python run_speech_recognition_seq2seq.py \
+    --model_name_or_path="openai/whisper-small" \
+	--dataset_name="mozilla-foundation/common_voice_11_0" \
+	--dataset_config_name="hi" \
+	--language="hindi" \
+	--eval_split_name="test" \
+ 	--gaudi_config_name="gaudi_config.json" \
+	--max_steps="5000" \
+	--output_dir="./results/whisper-small-clean" \
+	--gradient_accumulation_steps="2" \
+	--per_device_eval_batch_size="16" \
+	--logging_steps="25" \
+	--learning_rate="1e-5" \
+	--warmup_steps="500" \
+	--evaluation_strategy="steps" \
+	--eval_steps="1000" \
+	--save_strategy="steps" \
+	--save_steps="1000" \
+	--generation_max_length="225" \
+	--preprocessing_num_workers="16" \
+	--length_column_name="input_length" \
+	--max_duration_in_seconds="30" \
+	--text_column_name="sentence" \
+	--freeze_feature_encoder="False" \
+	--gradient_checkpointing \
+	--group_by_length \
+	--bf16 \
+	--overwrite_output_dir \
+	--do_eval \
+	--predict_with_generate \
+	--use_habana \
+	--use_hpu_graphs_for_inference
+```
```

Comment:
Let's keep only one example for inference, whichever you prefer

Reply:
Ok. Let's keep the 2nd one which uses the common_voice_11_0 dataset.